packages/zapp/console/src/components/Executions/Tables/NodeExecutionsTable.tsx

```diff
@@ -26,17 +33,47 @@ const scrollbarPadding = scrollbarSize();
  * NodeExecutions are expandable and will potentially render a list of child
  * TaskExecutions
  */
-export const NodeExecutionsTable: React.FC<NodeExecutionsTableProps> = ({
+export const NodeExecutionsTable: FC<NodeExecutionsTableProps> = ({
+  setSelectedExecution,
+  selectedExecution,
   abortMetadata,
-  nodeExecutions,
+  initialNodes,
 }) => {
-  const [selectedExecution, setSelectedExecution] = React.useState<NodeExecutionIdentifier | null>(
-    null,
-  );
+  const [nodeExecutions, setNodeExecutions] = useState<NodeExecution[]>([]);
   const commonStyles = useCommonStyles();
   const tableStyles = useExecutionTableStyles();
+  const nodeExecutionsById = useContext(NodeExecutionsByIdContext);
+
+  useEffect(() => {
+    if (nodeExecutionsById) {
+      const executions: NodeExecution[] = [];
+      // console.log('MYLOG initialNodes', initialNodes, nodeExecutionsById)
+      initialNodes.map((node) => {
+        if (nodeExecutionsById[node.scopedId]) executions.push(nodeExecutionsById[node.scopedId]);
+        else
+          executions.push({
+            closure: {
+              createdAt: dateToTimestamp(new Date()),
+              outputUri: '',
+              phase: NodeExecutionPhase.UNDEFINED,
+            },
+            id: {
+              executionId: {
+                domain: node.value?.taskNode?.referenceId?.domain,
+                name: node.value?.taskNode?.referenceId?.name,
+                project: node.value?.taskNode?.referenceId?.project,
+              },
+              nodeId: node.id,
+            },
+            inputUri: '',
+            scopedId: node.scopedId,
+          });
```

Comment:
if our linter doesn't complain about unwrapped statements, we should update it 😆 
```suggestion
        if (nodeExecutionsById[node.scopedId]) {
          executions.push(nodeExecutionsById[node.scopedId]);
        } else {
          executions.push({
            closure: {
              createdAt: dateToTimestamp(new Date()),
              outputUri: '',
              phase: NodeExecutionPhase.UNDEFINED,
            },
            id: {
              executionId: {
                domain: node.value?.taskNode?.referenceId?.domain,
                name: node.value?.taskNode?.referenceId?.name,
                project: node.value?.taskNode?.referenceId?.project,
              },
              nodeId: node.id,
            },
            inputUri: '',
            scopedId: node.scopedId,
          });
        }
```

Reply:
fixed